README.md

```diff
@@ -1,4 +1,4 @@
-# Preview of feature flags for VirtoCommerce Platform
+# Feature flags
```

Comment:
I think: "Feature management module" would be better

Reply:
Aligned